RocketChat/Rocket.Chat.ReactNative
# fix: show attachment with query parameters

## Proposed changes

Change the way that append rc_uid and rc_token to attachment's URL.

Before: https://mobile.rocket.chat/file-upload/E2WgFoTmcEGpHT9Kx/Group?32.png&rc_token=token&rc_uid=uid

After: https://mobile.rocket.chat/file-upload/E2WgFoTmcEGpHT9Kx/Group?32.png=&rc_token=token&rc_uid=uid

You can notice that at before has this `?32.png&rc_`, and now is `?32.png=&rc_`. With this new way, the method handles the 32.png as a new query param, not only concat strings as before.

## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/5121
## How to test or reproduce
There are 2 ways to test:
1 - Uploading a file
  - Using the web, upload an attachment (image, video) to any room and add to his name an `?`, Example: `Image?2.png`
  - Try to open that image or video using the mobile app, you should see the attachment

2 - Send a image URL with webhook that already has a query param
  - You can do the same as the user did on the issue #5121 

## Screenshots

  
 ## Types of changes




## Checklist



## Further comments

[TC-785]

[TC-785]: https://rocketchat.atlassian.net/browse/TC-785?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
